nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -127,6 +127,14 @@ let
           QEMU's swtpm options.
         '';
       };
+
+      hook = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = lib.mdDoc ''
+          QEMU's hook script.
```

Comment:
I think the documentation should cover the `PATH` gotchas mentioned [here](https://discourse.nixos.org/t/libvirt-installing-qemu-hook/385/8?u=tmplt).

Reply:
We can mention this but people using NixOS should know this. :sweat_smile:

Reply:
I don't consider "people should know this" a valid reason to omit useful documentation on details out of view from the user.